perf/perf_genericevents.py.data/raw_code.cfg

```diff
@@ -125,7 +125,7 @@ cache-misses = 0x0964
 branch-instructions = 0xc2
 branch-misses = 0xc3
 stalled-cycles-frontend = 0xa9
-ref-cycles = 0x120
+ref-cycles = 0x100000120
```

Comment:
@AYUSHJAIN951 is this change hold good for other architecture ? Any reference document  

@disgoel FYI

Reply:
This change is only for AMD arch
Link: https://lore.kernel.org/lkml/cover.1711352180.git.sandipan.das@amd.com/T/#m9c3ae61a5110d6ed81ca5c5d07e4807013bc0c6b

Reply:
> @disgoel FYI

changes looks good